gcloud-java-dns/README.md

```diff
@@ -300,12 +300,10 @@ while (recordIterator.hasNext()) {
 // Build and apply the change request to our zone if it contains records to delete
 ChangeRequestInfo changeRequest = changeBuilder.build();
 if (!changeRequest.deletions().isEmpty()) {
-  changeRequest = dns.applyChangeRequest(zoneName, changeRequest);
+  ChangeRequest pendingRequest = dns.applyChangeRequest(zoneName, changeRequest);
 
-  // Wait for change to finish, but save data traffic by transferring only ID and status
-  Dns.ChangeRequestOption option =
-      Dns.ChangeRequestOption.fields(Dns.ChangeRequestField.STATUS);
-  while (ChangeRequestInfo.Status.PENDING.equals(changeRequest.status())) {
+  // Wait for the change request to complete
```

Comment:
Should you also change the "When the change request is applied, it is registered with the Cloud DNS service for processing. We can wait for its completion as follow"... code snippet?